test/misc/ops-trust-proposal.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+
+import { describeContract, TestContext } from '../helpers/describe-contract';
+
+function init() {}
+
+describeContract('ops-trust-proposal', init, (ctx: TestContext) => {
```

Comment:
Is it possible to add a test that verifies the IPFS hash on the proposal was set correctly? It's fine to hardcode the proposal ID of 10 in the test, the test should look similar to [this one](https://github.com/dydxfoundation/governance-contracts/blob/master/test/safety-module/wind-down-safety-module.spec.ts#L25-L29).

Reply:
sounds good added with just the mock ipfs hash in the constants so it passes, let me know if that looks good!